src/spec/payid-protocol.md

```diff
@@ -116,11 +116,11 @@ This specification defines the PayID protocol - an application-layer protocol, w
 # JSON Format Design
   JSON, as described in [RFC8259][], defines a test format for serializing structured data. Objects are serialized as an unordered collection of name/value pairs. JSON does not define any semantics around the name/value pairs that make up an object. PayID protocol's JSON format defines name/value pairs that annotate a JSON object, property or array for PayID protocol resources.
 
-  The PayID client MUST request a PayID response in JSON format through the `Accept` header with the media type as defined below, optionally followed by format parameters. One of the optional parameters is the case-insensitive `q` value as described in Section 5.3.1 of [RFC7231][] to indicate relative preference.
+  The PayID client MAY request a PayID response in JSON format through the `Accept` header with the media type as defined below, optionally followed by format parameters. One of the optional parameters is the case-insensitive `q` value as described in Section 5.3.1 of [RFC7231][] to indicate relative preference.
```

Comment:
We're changing Accept headers from mandatory to optional, to support AWS S3 PayIDs.